### PR TITLE
Initial support for i18n

### PR DESCRIPTION
**Pull request by nickskyline**
:shipit:

### CODE REVIEW

File: app/views/application/_nav.haml
Comment (BGMP) [inline app/views/application/_nav.haml:1]: This needs to be updated to the latest navigation bar.

ref: https://github.com/Re-Volt-America/RVA/blob/master/app/views/application/_nav.haml

Comment (BGMP) [inline app/views/application/_nav.haml:1]: This needs to be updated to the latest navigation bar.

ref: https://github.com/Re-Volt-America/RVA/blob/master/app/views/application/_nav.haml

File: app/views/application/index.haml [hunk 1]
```diff
@@ -37,8 +42,8 @@
           .media-body.pb-3.mb-0.small.lh-125
             .d-flex.justify-content-between.align-items-center.w-100
               %strong.text-gray-dark
-                = "#{session_category_name(session)} Races"
+                = t("application.index.recent.races", category: session_category_name(session)) #{session_category_name(session)}
               .session-date
                 = pretty_datetime(session.date)
             %span.d-block.session-host
-              = "Hosted by #{session.host}"
+              = t("application.index.recent.hosted-by", host: session.host) #{session.host}
```
Comment (BGMP) [inline app/views/application/index.haml:49]: Missing a line break here

File: app/views/sessions/extra.haml
Comment (BGMP) [inline app/views/sessions/extra.haml:1]: What is this file?

**Review by BGMP — CHANGES_REQUESTED** [review]
Nice job! Take a look at the comments I left. We need those resolved before I can review further.

File: app/views/sessions/extra.haml
Comment (nickskyline) [reply app/views/sessions/extra.haml:1]: It was for testing purposes, I forgot to delete it

**Comment by BGMP** [comment]
You may request my review when you finish off the changes. Same goes for https://github.com/Re-Volt-America/RVA/pull/53

Thanks, Nick!

File: app/views/sessions/extra.haml
Comment (BGMP) [reply app/views/sessions/extra.haml:1]: Alright, I was so confused lol

**Comment by BGMP** [comment]
I think this is good enough to merge. The language selector is available at the user settings, it has its own tab. Although, I've been thinking that maybe bringing back the on-screen flag icon to change locales would be a good idea.

Could you try the changes out on your end @nickskyline ?

**Comment by nickskyline** [comment]
Looks and works good for me although it's a little weird to me that you have to introduce your password to only change the language.

And yes, bringing back that selector is great, specially for unregistered users.

**Comment by BGMP** [comment]
> Looks and works good for me although it's a little weird to me that you have to introduce your password to only change the language.
> 
> And yes, bringing back that selector is great, specially for unregistered users. @BGMP

The issue with unregistered users is keeping track of their locale. I'm not sure if rails-i18n supports local storage or a solution similar to that...

**Comment by BGMP** [comment]
Alright, I've brought back the language selector with the country flags and everything! We should be good to go now! 😄

**Pull request merged**